Makefile

```diff
@@ -5,7 +5,12 @@ test:
 configure:
 	pip install -e .'[dev]'
 
+docker_build_push:
+	docker build . -t hiredscorelabs/cornell:latest
+	echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
```

Comment:
Those were stored on the travis project settings of course, accessible only to master branch, using a token of our hiredscorelabs docker hub user.